website/client-old/js/components/groupApprovals/groupApprovalsController.js

```diff
@@ -9,7 +9,7 @@ habitrpg.controller('GroupApprovalsCtrl', ['$scope', 'Tasks',
     };
 
     $scope.approvalTitle = function (approval) {
-      return env.t('approvalTitle', {text: approval.text, userName: approval.userId.profile.name});
+      return env.t('approvalTitle', {type: approval.type, text: approval.text, userName: approval.userId.profile.name});
```

Comment:
By adding this piece in--type: approval.type-- we are able to access and display the task type on the groups.json file.